<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1023</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>931</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>177</width>
          <height>527</height>
         </rect>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>29</y>
           <width>160</width>
           <height>291</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="scroollLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <property name="documentMode">
          <bool>false</bool>
         </property>
         <property name="tabsClosable">
          <bool>true</bool>
         </property>
         <property name="movable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="tab">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <attribute name="title">
           <string>Tab 1</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="tab_2">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <attribute name="title">
           <string>Tab 2</string>
          </attribute>
          <widget class="QTableView" name="tableView">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>50</y>
             <width>561</width>
             <height>241</height>
            </rect>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3"/>
       </item>
       <item>
        <widget class="QLabel" name="labelState">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Предыдущая страница</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Следующая страница</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1023</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="createJournal"/>
    <addaction name="saveJournal"/>
    <addaction name="loadJournal"/>
    <addaction name="separator"/>
    <addaction name="changeJournal"/>
    <addaction name="separator"/>
    <addaction name="quitApp"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Предмет</string>
    </property>
    <addaction name="addSubject"/>
    <addaction name="delSubjects"/>
    <addaction name="separator"/>
    <addaction name="changeSubjectTimePeriod"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Ученик</string>
    </property>
    <addaction name="addSchoolBoy"/>
    <addaction name="delSchoolBoys"/>
    <addaction name="separator"/>
    <addaction name="changeSchoolBoy"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="Help"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="createJournal">
   <property name="text">
    <string>Создать журнал</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="saveJournal">
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="loadJournal">
   <property name="text">
    <string>Открыть</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="quitApp">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="addSubject">
   <property name="text">
    <string>Добавить предмет</string>
   </property>
  </action>
  <action name="changeJournal">
   <property name="text">
    <string>Изменить структуру таблицы</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Удалить учеников</string>
   </property>
  </action>
  <action name="addSchoolBoy">
   <property name="text">
    <string>Добавить ученика</string>
   </property>
  </action>
  <action name="delSchoolBoys">
   <property name="text">
    <string>Удалить учеников</string>
   </property>
  </action>
  <action name="Help">
   <property name="text">
    <string>Помощь</string>
   </property>
   <property name="toolTip">
    <string>Справка о функционале окна</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="delSubjects">
   <property name="text">
    <string>Удалить предметы</string>
   </property>
  </action>
  <action name="resSubject">
   <property name="text">
    <string>Итоги по предмету</string>
   </property>
  </action>
  <action name="resSchoolBoys">
   <property name="text">
    <string>Итоги по ученику</string>
   </property>
  </action>
  <action name="changeSubjectTimePeriod">
   <property name="text">
    <string>Изм. даты на промежутке</string>
   </property>
  </action>
  <action name="changeSchoolBoy">
   <property name="text">
    <string>Просмотр учеников</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>quitApp</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>466</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
